project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module

```diff
@@ -264,11 +264,9 @@ function c4m_features_og_group_dashboard_block_info() {
 function c4m_features_og_group_dashboard_block_view($delta = '') {
   $block = array();
 
-  if ($delta != 'c4m_cta_block') {
```

Comment:
Actually, we don't need that check as there is only one block left on the module.